### ISSUE TITLE
Feature Request: Hyperlinks

### ISSUE THREAD

**Issue by xnatski** [Automattic/simplenote-electron#2047]
Hey there team! I hope everything is awesome.

### What
As the title says, I'm here in hoping that we get a cool feature of linking notes through other notes; a concept of notes communicating with each other. I know you have tags; but it's not quite enough, I wish there were some way to just have **a hyperlink text** in my note that allows me to jump to other notes or keywords, when I click on them.

### Why

I've chosen simplenote as the software to handle my note taking needs as I've used different softwares in the past. it's very nice that you have some features that are not available elsewhere and you also have a wonderfully unique user interface. So I really love what you've done. I look forward in using simplenote for years to come.

That is why I hope to submit this suggestion of mine.

The feature suggestion idea was inspired from a blog post from "zattelkasten", a guide on how to Improve thinking and writing.

https://zettelkasten.de/posts/zettelkasten-improves-thinking-writing/

Please read it, you will learn a lot.

### How
this is exactly how I vision it to work:

tags can be written inside the note for easy referencing, if you've read the blog, you will know what I'm talking about.

Assuming you’re a writer or a thinker, why should you care about the way you take notes?

If you want to think creatively and write original articles and books, you need to form associations in your mind effectively. Notes can help you with that if you adhere to a few basic principles.

You can emulate communication processes with your own notes if you structure them in a certain manner. Notes can and should stimulate new associations and foster your creativity just like a good talk does. This is similar to how neurons in our brain work, so the note taking is almost like an extension of our brain.

Hypertext makes this possible, and full-text search (which we already have) should even make you faster than Luhmann, whose 20,000+ notes were stored on paper.

Basically what we need exactly is the ability to write and insert two different hyperlinks:
 1: hyperlink of a tag
 2: hyperlink of a keyword

Fundamentally, when you let notes point to each other using some kind of reference, you’re creating hypertext or hyperlink. This concept is well known on the web or docs where hypertext is presented as links you can click on in your browser.

On paper, you could write “see”, “cf.” or draw an arrow before you reference a note by number, arbitrary index, its title or whatever. The concept is identical to implementations of hyperlinks in a document on your computer or in its file system, where there are file aliases, hard links and symbolic links.

Be creative and really think about it, as long as you've read the blog, I'm sure you will figure it out.

**Comment by codebykat** [comment]
Hi @xnatski, thanks for this thoroughly-researched feature request! Awesome thinking on this one. Linking notes to each other (and links within notes) is something that's on our radar, but not on our immediate roadmap.

As you say, it's something we'll need to do a lot of thinking about, because it needs to work smoothly in every app, including the mobile apps. I'm closing this for now because we use Github only to track our active work, but we do track feedback when deciding what to work on in the future, so we will surely reference this. Thanks again!
